src/Command/UpdateCommand.php

```diff
@@ -27,8 +27,9 @@
  * For example,
  *
  * ```shell
- * ./yii migrate:up                                           # apply all new migrations
+ * ./yii migrate:up                                           # apply 10 new migrations
```

Comment:
Why 10? 
Apply all new migrations is expected behaviour for command `./yii migrate:up` without params.

Reply:
I think behavior should be the same as for `./yii migrate:new`

https://github.com/yiisoft/db-migration/blob/59f854689e684f961a008c4c219b02ab9668db80/src/Command/NewCommand.php#L24

Reply:
`new` is information command, that displays not yet applied migrations. In case, when there are many migrations, output all migrations will be non-comfortable. So by default better output limited count of new migration.

`up` mostly used for update database actual state. So by default better apply all migrations.

---

For example, `down` by default revert only one migration. because it is most popular case.